src/Microsoft.Management.UI.Internal/ManagementList/FilterCore/FilterRules/FilterRuleExtensions.cs

```diff
@@ -27,21 +27,7 @@ public static class FilterRuleExtensions
         /// </returns>
         public static FilterRule DeepCopy(this FilterRule rule)
```

Comment:
should we mark this obsolete so there is a compile time error

Reply:
done

Reply:
Removing the obsolete attribute here, since we cannot build with it marked obsolete.